src/js/Watermark.js

```diff
@@ -24,10 +24,8 @@ class Watermark extends Meister.ProtoPlugin {
         return packageJson.version;
     }
 
-    onPlayerPlaying() {
-        if (this.watermarkDrawn) return;
```

Comment:
Since playerCreated can in theory fire multiple times per instance as it's really more `playerPluginCreated` and not `meisterPlayerCreated`, I think this check should remain. If it does have to go then the `this.watermarkDrawn = false;` in the constructor should also go.

Reply:
It won't redraw the watermark on next playlist items if it's removed

Reply:
With `playerCreated` it won't redraw on new playlist items anyway, unless they use a different player type.